### PR TITLE
Homework14: Pavel Mitrafanau

### PR DESCRIPTION
**Pull request by Pavel2812mi**
Good day, Sergei
My homework14 files are ready for review

### CODE REVIEW

File: Homework14_1.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+"""files"""
+
+
+import os
+
+if not os.path.exists("D:\\students.txt"):
+    with open("D:\\students.txt", "w", encoding="utf-8") as file:
```
Comment (Stml89) [inline Homework14_1.py]: Please do not use absolute path, because it will work for but not for other ppl: https://archive.ph/0aXb1

**Review by Stml89 — CHANGES_REQUESTED** [review]

**Comment by Pavel2812mi** [comment]
Absolute file path has been replaced with relative path, now should be fine

File: Homework14_1.py
Comment (Stml89) [inline Homework14_1.py:1]: Is that possible to use function(s) here?

**Review by Stml89 — CHANGES_REQUESTED** [review]

**Comment by Pavel2812mi** [comment]
Improved the code in the module by using functions
I also updated the ruff linter launch command.

**Review by Stml89 — APPROVED** [review]

**Review by Stml89 — APPROVED** [review]

**Pull request merged**